core/src/main/scala/org/apache/spark/executor/ShuffleWriteMetrics.scala

```diff
@@ -17,8 +17,12 @@
 
 package org.apache.spark.executor
 
+import java.{lang => jl}
```

Comment:
the convention inside spark is to rename the java box classes w/ prefix"J", eg. `JLong`: https://demo.fluentcode.com/source/spark/master/master/sql/hive/src/main/scala/org/apache/spark/sql/hive/client/HiveShim.scala?squery=JLong#L20